RunTests.cmd

```diff
@@ -8,43 +8,95 @@ call :pauseOnError msbuild -t:Clean
 :: CREATE A NUGET PACKAGE WITH BELOW COMMAND AND ADD TO LOCAL FOLDER + UPDATE NUGET CONFIG FILE TO READ FROM THAT LOCATION
 :: msbuild -p:configuration=Release
 
+:: Based on `dotnet test` documentation, `Platform` propertry has no effect on choosing the underlating architecture.
+:: It's required running tests by desired `dotnet` version for a platform(x64, x86, Arm64) specific assessment.
```

Comment:
I think this is what you mean:
```suggestion
:: Based on `dotnet test` documentation, the `Platform` property has no effect on choosing the underlying architecture for the test execution environment.
:: You need to install and run the `dotnet` command for a specific architecture (x64, x86, Arm64).
```
But then I'm not sure how the below commands are meant to work. When a `dotnet` command below specifies `-p:Platform="Win32"`, what is that meant to do? Should we also be targeting `dotnet` directly to `%ProgramFiles(x86)%\dotnet\dotnet.exe` in the command? I see `DotnetPath` has been introduced below. Should we have some smart defaults in build.proj to point it to a platform-specific instance of dotnet.exe when Platform is specified?

Reply:
The `Platform` properly belongs to MSBuild and `dotnet test` command forwards it to MSBuild, for building and resolving the targeted assembly.